sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/aio/_operations/_patch.py

```diff
@@ -29,6 +33,50 @@
 ClsType = Optional[Callable[[PipelineResponse[HttpRequest, AsyncHttpResponse], T, Dict[str, Any]], Any]]
 
 
+class AsyncAnalyzeDocumentLROPollingMethod(AsyncLROBasePolling):
```

Comment:
same comments as sync apply to async for this file

Reply:
Updated same in async